docs/source/installation.rst

```diff
@@ -24,33 +24,33 @@ Installation with PyPi
 
 In a terminal, issue the following command::
 
-    $ pip install fury
+    pip install fury
 
```

Comment:
remove the "::" and replace the command with:

.. code-block:: shell

  pip install fury

Reply:
thank you @Nibba2018  got to learn about rst too. I request you to have a review and some feedbacks about it